src/org/ods/OdsContext.groovy

```diff
@@ -436,6 +440,10 @@ class OdsContext implements Context {
     config.displayNameUpdateEnabled = displayNameUpdateEnabled
   }
 
+  String getDockerDir() {
+    return config.dockerDir
+  }
```

Comment:
I think you'll need to add this to the `Context` interface too.

Reply:
done.